README.md

```diff
@@ -2,20 +2,14 @@
 This package provides an HTTP Client library built on top of SwiftNIO.
 
 This library provides the following:
-- First class support for Swift Concurrency (since version 1.9.0)
+- First class support for Swift Concurrency
```

Comment:
I was tempted to suggest leaving the version numbers here but seeing as the inclusion code snippet before uses `from: "1.9.0"` it feels consistent to drop it.